### PR TITLE
Troubleshoot with PBID on the gateway computer

### PR DESCRIPTION
**Pull request by greggyb**
It can often be useful to troubleshoot source connectivity issues for a gateway by installing Power BI Desktop on the gateway computer and testing connectivity there.

Add a brief description of this troubleshooting technique.

I recommend this often to clients and those asking for help with gateway connections, and it is usually a helpful troubleshooting step.

### CODE REVIEW

File: data-integration/gateway/service-gateway-tshoot.md [hunk 1]
```diff
@@ -124,6 +124,16 @@ To find the event logs for the *on-premises data gateway service*, follow these
 
 ![On-premises data gateway event logs.](media/service-gateway-tshoot/on-prem-data-gateway-event-logs.png)
 
+## Troubleshooting refresh failures for a specific source
+
+Refreshes utilizing the gateway require that the source be accessible from the computer with the gateway installation.
+As such, when troubleshooting a data source, especially one that requires additional components to be installed on the computer (such as a third party database driver) or additional configuration (such as access to a network drive file or folder), it can be helpful to troubleshoot with Power BI Desktop from the gateway computer.
```
Comment (JasonWHowell) [inline data-integration/gateway/service-gateway-tshoot.md:130]: The sentence is too long for translation. Simpler is better - split it into two.

```suggestion
To troubleshoot a data source issue, use Power BI Desktop locally on the gateway computer to test the connection. This test is especially helpful if the data source requires additional components to be installed on the computer, such as a third-party database driver. Also, a local test helps narrow down if the data source connection requires additional environment settings or access to a shared network drive file or folder.
```

File: data-integration/gateway/service-gateway-tshoot.md [hunk 2]
```diff
@@ -124,6 +124,16 @@ To find the event logs for the *on-premises data gateway service*, follow these
...
+Using Power BI Desktop on the gateway computer allows for more rapid iterations of testing the connection and configuring any requirements on the gateway computer directly.
```
Comment (JasonWHowell) [inline data-integration/gateway/service-gateway-tshoot.md:131]:
```suggestion
This technique allows you to test iteratively, testing the connection on the gateway computer after each configuration change.
```

**Review by JasonWHowell — COMMENTED** [review]
Thanks for the changes. Making some editorial suggestions if it doesn't change the meaning too much.

**Comment by JasonWHowell** [comment]
@DougKlopfenstein I agree with these changes, having been a troubleshooter for many years. Do you want to review as well?

File: data-integration/gateway/service-gateway-tshoot.md [hunk 3]
```diff
@@ -124,6 +124,17 @@ To find the event logs for the *on-premises data gateway service*, follow these
...
 ![On-premises data gateway event logs.](media/service-gateway-tshoot/on-prem-data-gateway-event-logs.png)
 
+## Troubleshoot refresh failures for a specific source
+
+Refreshes utilizing the gateway require that the source be accessible from the computer with the gateway installation.
+To troubleshoot a data source issue, use Power BI Desktop locally on the gateway computer to test the connection.
+This test is especially helpful if the data source requires additional components to be installed on the computer, such as a third-party database driver. Also, a local test helps to test data source connections that require additional environment settings, such as access to a shared network drive file or folder.
+This technique allows you to test iteratively, testing the connection on the gateway computer after each configuration change.
+
+Although it is not a guarantee of a successful refresh through the gateway, a successful Power BI Desktop refresh from the gateway computer is a strong indicator that everything is configured correctly on the gateway computer.
```
Comment (DougKlopfenstein) [inline data-integration/gateway/service-gateway-tshoot.md:134]: Microsoft style requires the use of contractions (for more friendly text), so change "is not" to "isn't".

File: data-integration/gateway/service-gateway-tshoot.md [hunk 4]
```diff
@@ -124,6 +124,17 @@ To find the event logs for the *on-premises data gateway service*, follow these
...
+In other words, if you cannot refresh in Power BI Desktop from the gateway computer, it is highly unlikely that a refresh through the gateway will succeed.
```
Comment (DougKlopfenstein) [inline data-integration/gateway/service-gateway-tshoot.md:135]: Again, use contractions - change "cannot" to "can't" and "it is" to "it's".

**Comment by DougKlopfenstein** [comment]
@greggyb requested some minor Microsoft style changes.

File: data-integration/gateway/service-gateway-tshoot.md (hunk 3 above)
Comment (greggyb) [reply data-integration/gateway/service-gateway-tshoot.md]: "is not" -> "isn't" in latest commit

**Review by DougKlopfenstein — APPROVED** [review]

**Pull request merged**